src/components/ListPage/ListPage.ts

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { RouteComponentProps } from 'react-router';
+import * as MessageCenter from '../../utils/MessageCenter';
+
+export namespace ListPage {
+  export interface URLParameter {
+    name: string;
+    value: string;
+  }
+
```

Comment:
This should be in the types folder, maybe we'll need this in another views. What do you think?